gen_redirects.py

```diff
@@ -52,12 +52,6 @@
 # Untemplated redirects
 REDIRECTS = {
     "/api/": "/reference/",
-    "/api/backends/": "/reference/backends/",
-    "/api/backends/base/": "/reference/backends/base/",
-    "/api/backends/pandas/": "/reference/backends/pandas/",
```

Comment:
any reason to remove the pandas redirect?

Reply:
Again, maybe it should be `"/backends/pandas/"` instead then.